### PR TITLE
[GR-51625] Document debugging native image agent

### PR DESCRIPTION
**Pull request by galderz**
Document how to debug the native image agent. FYI @fniephaus @peter-hofer

### CODE REVIEW

File: substratevm/src/com.oracle.svm.agent/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash
+$ gdb java
+```
+
+Then set the agent arguments for the java process:
+
+```bash
+(gdb) set args -agentlib:native-image-agent=config-output-dir=./native-image-config -jar app.jar
+```
```
Comment (zakkak) [inline substratevm/src/com.oracle.svm.agent/README.md:25]: FYI you can do this in one line with:

```
gdb --args java -agentlib:native-image-agent=config-output-dir=./native-image-config -jar app.jar
```

Comment (galderz) [reply substratevm/src/com.oracle.svm.agent/README.md:25]: Yeah I was aware. Do you prefer that style?

Comment (zakkak) [reply substratevm/src/com.oracle.svm.agent/README.md:25]: In my workflow certainly, for the guide I don't really mind, either is fine.

Comment (galderz) [reply substratevm/src/com.oracle.svm.agent/README.md]: Fair enough. If you find it more convenient with your workflow let's go with that. I've updated the PR.

**Review by fniephaus — APPROVED** [review]
LGTM, thanks!

**Comment by fniephaus** [comment]
Thanks for the contribution. This is merged via #8259.

**Pull request closed without merging**